AssetManagementTestProject/PageObj/LoginPage.cs

```diff
@@ -19,4 +19,14 @@ public void Login(string userName, string password)
         TakeScreenShot();
         Click(btnLogin);
     }
+
+    public List<string> LoginPageElementUI()
```

Comment:
I think method name should start with a verb, not a noun here

Reply:
1. This method name should be `GetSmt()`
2. This method has no meaning if you use it to assert some things on the screen. It check for only the `tfUsername`

Reply:
Done